info.limpet.stackedcharts.ui/src/info/limpet/stackedcharts/ui/editor/commands/MoveAxisCommand.java

```diff
@@ -8,33 +8,54 @@
 
 public class MoveAxisCommand extends Command
```

Comment:
@theanuradha - I've played around with the logic here, to tidy the code. It works for me, and I'll do the merge. But, please could you double-check that MoveAxis and MoveChart still work correctly?